demos/risk/InfrastructureRisk/WaterSupplyNetwork/exposure_demo_edges.csv

```diff
@@ -0,0 +1,24 @@
+id,lon,lat,taxonomy,number,vulnerable,type,start_node,end_node,supply_or_demand,length,parent_id
```

Comment:
@asthapoudel did you want to change the field name here from "supply_or_demand" to something else? Please recommend a suitable replacement.

Reply:
Can we have two options to define this field??both "supply_or_demand" or "Function"

Reply:
Remember that "function" is very close to "functional" which appears in multiple places in the code, and can cause confusion down the line. Also, do we need both? It's preferable to have one term that generally covers all cases.

Reply:
Yes, function is very close to functional but the most appropriate too.
What do you think about "purpose"/"NodeRole"/ "Usage"??
 It is just that I want to keep both just for giving more options to the users.

Reply:
Please be aware of this: https://github.com/asthapoudel/oq-engine/blob/master/openquake/calculators/event_based_damage.py#L290
The connectivity analysis is currently done if supply_or_demand is in assetcol.array.dtype.names.
Perhaps 'function' is a bit too generic and it risks to overlap with something else?

Reply:
We changed the approach, adding a new parameter to the ini file and renaming `supply_or_demand` to `purpose`